src/react-components/webrtc-screenshare-dialog.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import DialogContainer from "./dialog-container.js";
+
+export default class WebRTCScreenshareDialog extends Component {
```

Comment:
Since this dialog is shown when screen sharing is unsupported, the copy should probably actually say something along the lines of "Your browser appears to not support screen sharing". Also the component name should maybe be something like `WebRTCScreenshareUnsupportedDialog`.